integration

```diff
@@ -149,6 +149,7 @@
   "ben8p/home-assistant-bunq-balance-sensors",
   "benjamin-dcs/file-plusplus",
   "benleb/sureha",
+  "benlikethecolor/HomeAssistant-WMATA",
```

Comment:
As this is only relevant for users in a specific country, this needs a country key in the released hacs.json file.